src/test/java/com/trello/trello/stepdefs/TeamSteps.java

```diff
@@ -0,0 +1,60 @@
+package com.trello.trello.stepdefs;
+
+import com.trello.trello.entities.Context;
+import com.trello.trello.ui.pages.TeamPage;
+import com.trello.trello.ui.pages.components.LeftMenuComponent;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import java.util.Map;
+
+/**
+ * TeamSteps class.
+ */
+public class TeamSteps {
+
+    private Context context;
+    private LeftMenuComponent leftMenu;
+    private TeamPage teamPage;
+
+    /**
+     * Inject the Context class.
+     *
+     * @param cont
+     */
+    public TeamSteps(final Context cont) {
+        this.context = cont;
+    }
+
+    /**
+     * The method created a leftMenuComponent instance.
+     */
+    @When("I navigate to create team button")
+    public void iNavigateToCreateTeamButton() {
```

Comment:
It would be better if you use `navigate `when you use the page transporter and `go ` when you go to the page interacting with the UI